docs/pi_imaging.md

```diff
@@ -198,5 +207,6 @@ To keep the clean state of the image, remove any SSH keys
 and command history.
 ```
 rm ~/.ssh/authorized_keys
+rm ~/amplipi-dev/house.json*
```

Comment:
This should be handled by the (cleanup)[https://github.com/micro-nova/AmpliPi/blob/main/scripts/cleanup] script that Mahdee runs at checkout, but can't hurt here either.